orc8r/gateway/c/common/sentry/SentryWrapper.cpp

```diff
@@ -157,6 +172,8 @@ void sentry_log_error(const char* message) {
 
 #else
 
+sentry_config_t construct_sentry_config_from_mconfig() {}
```

Comment:
Can we silence these warnings?

Reply:
I think you can return an empty config instead of disabling the linter:
```cpp
sentry_config_t construct_sentry_config_from_mconfig() {
    return {0};
}
```